oncotator/utils/FieldMapCreator.py

```diff
@@ -0,0 +1,111 @@
+from oncotator.utils.ConfigUtils import ConfigUtils
+
+
+class FieldMapCreator(object):
+    """Generic class for creating a mapping of field name to an annotation name.  Under the hood this is a simple dict.
+    """
+
+    @staticmethod
+    def choose_best_annotation(header, m, alternative_dict, deprioritize_input_annotations):
+        """  Choose the best annotation in mut for the given header.
+        :param header:
+        :param m:
+        :param alternative_dict:
+        :param deprioritize_input_annotations:
+        :return: if no suitable annotation, None
+        """
+
+        annotation_names = sorted(list(set(m.keys() + m.getAttributeNames())))
+        hdr = header.lower()
+        alternatives = alternative_dict.get(hdr, [])
+        full_possibilities = [header] + alternatives
+        result = None
+
+        if not deprioritize_input_annotations:
+            for p in full_possibilities:
+                if p in annotation_names:
+                    result = p
+                    break
+        else:
+            second_choice = None
+            is_choosing_second_choice = True
+            for p in full_possibilities:
+                if p in annotation_names and m.getAnnotation(p).getDatasource() != "INPUT":
```

Comment:
Why not just return early if you find the right value instead of the more complicated `is_choosing_second_choice` flag?  Is there an oncotator policy of avoiding multiple return statements?